2018-08-10 11:11 INFO    : Object with name data_plot_hist exists, replacing (object_sharer.py:138)
